index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Assignment_01 </title>
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+        <!---- <script src="jquery-3.4.1.min.js"></script>-->
+        <script src="myscript.js"> </script>
+        
+    </head>
+    <body>
+
+        <!--     
+            Deepa Natarajan
+            assignment01.htm
+            INFO_2340_WW
+            Ms.Thoendel
+            12/08/2019
+        -->
```

Comment:
Super happy to see this comment block!

Reply:
I am super happy too, to see all your comments. It is very lively, I love this class structure very much.Thank you!!